guides/user/get-started/index.md

```diff
@@ -7,13 +7,53 @@ sidebar:
 
 {% include toc %}
 
+This article is for Spinnaker *users*. *Operators* of Spinnaker can start with
+[Set up Spinnaker](/setup/).
+
+This article assumes you are new to Spinnaker. It contains basic information
+about what Spinnaker does and what you do to get Spinnaker to do it.
+
+Spinnaker deploys services to one or more cloud providers. It
+
+
+<!--move this -->
+## A note about inferred applications
+
+If you've deployed Spinnaker on a Kubernetes cluster, you might several
+application already available
```

Comment:
" ... might several application already available" seems to be a sentence fragment.